frontend/app/src/lib/CodeExamples.ts

```diff
@@ -0,0 +1,22 @@
+import { CodeExample } from "@/domain/CodeExample";
+
+export const codeExamples: CodeExample[] = [
+	{
+		title: "Dwarf",
+		shortDescription:
+			'The dwarf "bombs" the core at regular intervals with DATs while making sure to not hit itself',
+		code: "add.ab  #4, 3\nmov.i   2, @2\njmp     -2\ndat     #0, #0",
```

Comment:
Minor nitpick, but can you add a newline at the end of the code examples? Looks a bit nicer in the editor then and you can immediately start editing on a fresh line

Reply:
:nerd_face:

Reply:
will be done :)

Reply:
done :+1: